go/src/koding/klientctl/metrics/metrics.go

```diff
@@ -0,0 +1,76 @@
+package metrics
+
+import (
+	"context"
+	"path/filepath"
+
+	kitesconfig "koding/kites/config"
+
+	statsd "github.com/DataDog/datadog-go/statsd"
+)
+
+// Metrics wraps metric
+type Metrics struct {
+	bolt    *BoltQueue
+	Datadog *statsd.Client
+}
+
+// New creates new metrics collector.
+func New() (*Metrics, error) {
+	boltPath := filepath.Join(kitesconfig.KodingHome(), "metrics.bolt")
```

Comment:
```
boltPath := configstore.CacheOptions("metrics").File
```

Otherwise events produced in devel environment could get pushed to production and vice-versa. This can happen when e.g. I'd be developing against my devel koding, then `kd auth` to production and after `sudo kd restart` klient is going to pick up shared metrics.bolt.

The configuration produced by configstore makes the database file paths unique per baseurl.